app/lib/features/chat/widgets/rooms_list.dart

```diff
@@ -65,26 +65,17 @@ class _RoomsListWidgetState extends ConsumerState<RoomsListWidget> {
     String title = AppLocalizations.of(context)!.chat;
 
     if (ref.watch(hasRoomFilters)) {
-      String subFiltered = 'chats';
       final selection =
           ref.watch(roomListFilterProvider.select((value) => value.selection));
       switch (selection) {
         case FilterSelection.dmsOnly:
-          subFiltered = 'DMs';
+          title = 'DMs';
           break;
         case FilterSelection.favorites:
-          subFiltered = 'Bookmarked chats';
+          title = 'Favorites';
```

Comment:
nope. We call that "bookmarked" in Acter. Favorites implies a value judgement of love that isn't necessarily true nor feels correct. Bookmark is a more neutral term.


```suggestion
          title = 'Bookmarked';
```